src/aprl/configs/multi/score.py

```diff
@@ -1,138 +1,155 @@
-            return None
-        else:
-            return adversary_type, os.path.abspath(path)
+def _gen_configs(victim_fns: Iterable[AgentConfigGenFn],
+                 opponent_fns: Iterable[AgentConfigGenFn],
+                 envs: Optional[Iterable[str]] = None,
+                 ) -> List[EnvAgentConfig]:
+    """Helper function to generate configs.
 
-    return _gen_configs(victim_fn=_zoo_identity, adversary_fn=adversary_fn, **kwargs)
+    :param victim_fns: list of callables, taking environment name and agent index.
+    :param opponent_fns: as above.
+    :param envs: optionally, a list of environments to generate configs for.
 
+    :return A list of (env, agent_a_type, agent_a_path, agent_b_type, agent_b_path).
+    """
+    if envs is None:
+        envs = BANSAL_GOOD_ENVS
 
-PATHS_AND_TYPES = 'env_name:agent_a_type:agent_a_path:agent_b_type:agent_b_path'
+    configs = []
+    for env in envs:
+        victim_index = VICTIM_INDEX[env]
+        victims = list(itertools.chain(*[fn(env, victim_index) for fn in victim_fns]))
+        opponents = list(itertools.chain(*[fn(env, 1 - victim_index) for fn in opponent_fns]))
+
+        for victim_type, victim_path in victims:
+            for adversary_type, adversary_path in opponents:
+                if victim_index == 0:
+                    cfg = EnvAgentConfig(env, victim_type, victim_path,
+                                         adversary_type, adversary_path)
+                elif victim_index == 1:
+                    cfg = EnvAgentConfig(env, adversary_type, adversary_path,
+                                         victim_type, victim_path)
+                else:
+                    raise ValueError(f"Victim index '{victim_index}' out of range")
 
+                if cfg not in configs:
+                    configs.append(cfg)
 
-def _summary_paths():
-    summary_agents = {
-        # ([median victim Zoo id], [best Zoo opponent ID]) -- note 0-indexed
-        'multicomp/KickAndDefend-v0': ([2], [2]),
-        'multicomp/YouShallNotPassHumans-v0': ([1], [1]),
-        'multicomp/SumoHumansAutoContact-v0': ([2], [3]),
-    }
-    adversary_agents = {env: (victim_ids, victim_ids)
-                        for env, (victim_ids, _opponent_ids) in summary_agents.items()}
-    adversaries = _adversary_vs_victims('ppo2', get_adversary_paths(),
-                                        no_transfer=True, agents=adversary_agents)
-    zoo = _env_agents(agents=summary_agents)
-    return adversaries + zoo
+    return configs
 
 
 def make_configs(multi_score_ex):
+
+    # ### Modifiers ###
+    # You can use these with other configs.
+
+    # Accuracy
+
     @multi_score_ex.named_config
-    def high_accuracy(exp_name, score):
+    def high_accuracy(score):
         score = dict(score)
         score['episodes'] = 1000
         score['num_env'] = 16
-        exp_name = 'high_accuracy_' + exp_name
+        exp_prefix = {'high_accuracy': 'high_accuracy'}  # noqa: F841
```

Comment:
Add documentation at least here, plausibly also in other places, explaining that only keys of this dictionary are ever used

Reply:
I've set the values to all be `None` to self-document that they're not very important, and added a comment under modifiers to this effect.